{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -38,4 +40,4 @@
         TemplateView.as_view(template_name='index.html'),
         name='frontend',
     ),
-]
+] + websockets.routes
```

Comment:
This might merit an explanatory comment, as it is a bit unorthodox. Something like:

```
So that non-HTTP paths can be accessible by `reverse` in Python and `window.api_urls` in JavaScript. These will usually only be the path component, not a full URL, and so the caller will have to build them with the right scheme and authority sections.
```